Added Sample Channels file for UK Virgin TIVO (V6)

asantaga opened the pull request:
Added Sample file for UK Virgin Tivo (V6 Model but I dont think the channels change on different tivos.

To generate the channels I wrote the following python (fancied trying python out :-) ) let me know if you want me to include this in the pull or we could rewrite it in JS

Sample Code in in forked repo https://github.com/asantaga/alexa_tivo_control/blob/master/channelGenerators/generateUKTivoChannels.py

asantaga committed:
Added Sample Channels file for UK Virgin TIVO (V6)

jradwan commented:
Can you paste your python code formatted so I can see the indents and try it out for myself? That could be a useful tool.

asantaga committed:
Added Python based channels.json generator code

asantaga commented:
Re Channel Code generator
- Added to my code, and put it in a sub director called channelGenerators..

I think its only really useful as a "head" start to generate the channel listings and then you need to tweak the listings..

e..g BBC One becomes B.B.C 1

Also 
To get your code working for my tivo I removed a whole chunk of utterences because they didnt work in the UK. I was going to look at them again sometime soon (now that wifes done her shopping)

Additionally, the UK tivo box is called a V6 Box, not sure which model this relates to www.virginmedia.com/shop/tv/v6.html

Alas Ive just started learning the Alexa APi but do let me know how I can help

jradwan commented:
Very cool ... thanks for the python code and the sample UK Virgin file. I'll merge this PR. I think I'll move all of the "example" json files into a separate subdirectory to keep them separate too. That way as we add more examples they don't clutter up the root directory.
